### DISCUSSION TITLE
voltstorage : Can I connect to voltstorage?

### DISCUSSION THREAD

**Discussion by lindstroem165** [grafana/grafana-infinity-datasource#222] [Specific APIs]
Hi all, I am absolutely willing to read all the documentation, but probably you guys can save me a lot of time. Is generally possible to connect to the voltstorage api? The documentation is here: https://api-reference.voltstorage.com/

It is basically a rest api, so as much as I understand this should work. Are there any limitations?

thanks in advance!
Nicklas

**Comment by yesoreyeram** [comment]
I don't know much about voltstorage. If it is a rest api with basic authentication/header/query based authentication, it should work. Let me know what is not working for you. If it is working, feel free to share how it set it up so that others can get benefit from it.
